pkg/keccak/Makefile

```diff
@@ -0,0 +1,38 @@
+PKG_NAME	:= keccak
+PKG_URL		:= https://github.com/adrianghc/KeccakCodePackage
+PKG_VERSION	:= 84fd5833efc8e5d0d4e1edaa2595dd2f6f160c13
+
+.PHONY: all
+
+all: git-download
+ifeq ($(CPU),native)
```

Comment:
So you mean e.g. instead of "@cp $(PKG_BUILDDIR)/Build/CompilerFlagsCortexM0.build $(PKG_BUILDDIR)/CompilerFlags.build" and replacing the name of the .build file for each recipe I should just do a generic "@cp $(PKG_BUILDDIR)/Build/$(CPU_ARCH).build $(PKG_BUILDDIR)/CompilerFlags.build" (or similar) once?

Reply:
@kaspar030, could we resolve this?

Reply:
I meant sth like:

```
ifeq ($(CPU),native)
  CPU_ARCH_NAME:=Native
  APU_ARCHIVE_NAME:=Native
else
ifeq ($(CPU),native)
  CPU_ARCH_NAME:=CortexM
  CPU_ARCHIVE_NAME:=ARMv6M
...
endif
```

Then have only one recipe using those variables:

```
all: git-download
 @cp $(PKG_BUILDDIR)/Build/CompilerFlags$(CPU_ARCH_NAME).build $(PKG_BUILDDIR)/CompilerFlags.build"
  ...
  $(MAKE) KeccakHash800$(CPU_ARCHIVE_NAME).a -C $(PKG_BUILDDIR)
  ...
```

That way the long ifdef only contains the (hopefully) minimal differences, and the recipe is only spelled out once. You'll thank yourself the next time you need to change anything there, or add another architecture.

Reply:
Thank you very much for your feedback, this should be addressed now.